Cesium.CodeGen.Tests/CodeGenMethodTests.cs

```diff
@@ -361,10 +361,11 @@ int main()
 }");
 
     [Fact]
-    public void InvalidPointerWithIntSubtractionTest() => DoesNotCompile(@"int main() {
+    public void InvalidPointerWithIntSubtractionTest() => DoTest(@"int main() {
```

Comment:
_(Minor)_ The name of this test is now misleading: it is no longer `Invalid`, right?